pg/seed.pgsql

```diff
@@ -21,22 +21,29 @@ SET default_tablespace = '';
 SET default_table_access_method = heap;
 
 --
--- Name: tasks; Type: TABLE; Schema: public; Owner: postgres
+-- Name: animals; Type: TABLE; Schema: public; Owner: tpl1_2021h1
 --
 
-CREATE TABLE public.tasks (
+CREATE TABLE public.animals (
     id integer NOT NULL,
-    name text
+    name character varying,
+    age character varying,
+    weight character varying,
+    gender character varying,
+    breed character varying,
+    blurb character varying,
+    foster boolean,
+    picture_link character varying
 );
 
 
-ALTER TABLE public.tasks OWNER TO postgres;
+ALTER TABLE public.animals OWNER TO tpl1_2021h1;
```

Comment:
For the pg_dump, if you wanted to omit the owner you can pass in a ```--no-owner``` flag after your table names

Ex for mine:
```pg_dump -t 'pfaf' -t 'trefle' -t 'plants' --no-owner postgres > create_plants.sql```